DEVELOP.md

```diff
@@ -0,0 +1,57 @@
+# Davidia development
+
+To get started with Davidia development, create a conda environment called `davidia`
+
+### `conda env create --file environment.yml`
+
+Activate it:
+
+### `conda activate davidia`
+
+Install Typescript dependencies (read pnpm's [installation guide](https://pnpm.io/installation), if needed)
+
+### `pnpm install`
+
+Build web client
+
+### `pnpm build`
+
+## Running Python plot server
+
+From the top level of the repository, you can run:
+
+### `cd server && uvicorn --factory davidia.main:create_app` or `PYTHONPATH=server python server/davidia/main.py -c`
```

Comment:
`davidia.main:run_app`

Reply:
Fixed `main.py` to reinstate `create_app`